sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkGetColumnsOperation.scala

```diff
@@ -141,17 +174,17 @@ private[hive] class SparkGetColumnsOperation(
           column.name, // COLUMN_NAME
           toJavaSQLType(column.dataType.sql).asInstanceOf[AnyRef], // DATA_TYPE
           column.dataType.sql, // TYPE_NAME
-          null, // COLUMN_SIZE
+          getColumnSize(column.dataType).map(_.asInstanceOf[AnyRef]).orNull, // COLUMN_SIZE
```

Comment:
Could you make the title and description clearer? e.g., which field does this PR add (COLUMN_SIZE, DECIMAL_DIGITS, and NUM_PREC_RADIX, and ORDINAL_POSITION?)

Reply:
OK, thanks Takeshi ~